### PR TITLE
React set-up

### PR DESCRIPTION
**Pull request by gasangw**
- The easiest way to create a React application is using Create React App (CRA) and following the instructions in this [guide](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
- The application should run in the browser without errors.

### CODE REVIEW

**Review by RolandM99 — COMMENTED** [review]
Hi @gasangw, 

While you made a great effort in this project unfortunately I cannot proceed to review your code.

## Invalid Code Review Request ❌

You have submitted a project, that has failed for some Linters check. Linter result should be green, meaning the last commit should have a green check mark (✅). Kindly, fix this issue and then ask for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (TedLivist) [inline README.md]: - [x] Kindly format your entire README using the [Microverse template](https://github.com/microverseinc/readme-template)

  Remember to include the specific steps on how to set up and run the app locally.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @gasangw,

Good job so far!
✔️ Nice design
✔️ No errors
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+- HTML, CSS & JavaScript
+- React components.
+- React props.
+- React state
+
+## Getting Started
+
+- npx create-react-app my-app
+- cd my-app
+- npm start
```
Comment (DammyShittu) [inline README.md:19]: - **[OPTIONAL]** The previous reviewer meant that you should include steps on how to set up this very app, not create a new react app. See example
  ```
  - git clone https://github.com/gasangw/Math-Magicians.git
  - cd Math-Magicians
  - npm install
  - npm start
  ```

**Review by DammyShittu — APPROVED** [review]
Hi @gasangw,

## APPROVED 🟢

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (gasangw) [reply README.md:19]: Thank you so much @DammyShittu

**Pull request merged**